app/models/pacbio/well.rb

```diff
@@ -25,10 +26,6 @@ def position
       "#{row}#{column}"
     end
 
-    def position_leading_zero
-      "#{row}#{column.rjust(2, '0')}"
-    end
-
     def summary
       "#{sample_names},#{comment}"
```

Comment:
This is the methods that gives us the trailing `,` in run comments. We could add that as an issue to fix another time? 🙂

Reply:
👍 #258